### ISSUE TITLE
[Question] How to predict on the test set with known labels?

### ISSUE THREAD

**Issue by ihopethiswillfi** [timeseriesAI/tsai#241]
Hi, I am using 'test' as a holdout set to basically double-check my performance on the validation set.
I'm using the following code:
```
splits = get_splits(y, n_splits=1, valid_size=0.1, test_size=0.05)
dsets = TSDatasets(X, y_labels, tfms=[None, [Categorize()]], splits=splits, inplace=True)
dls = TSDataLoaders.from_dsets(dsets.train, dsets.valid, ...)
```
So while dsets.train and dsets.valid exist, there is no dsets.test.
How do I predict on test.X and check accuracy against test.y ?
I'm sure it's very simple, but after reading examples and even diving into the code, I was still unable to figure this out.

Any help much appreciated!
